Sources/OpenGoogleSignInSDK/OpenGoogleSignInSDK.swift

```diff
@@ -199,6 +237,16 @@ public final class OpenGoogleSignIn: NSObject {
         
         return request
     }
+    
+    /// Returns `URLRequest` to retrieve user's profile data
+    private func makeProfileRequest(user: GoogleUser) -> URLRequest? {
+        guard let profileURL = OpenGoogleSignIn.profileURL else { return nil }
+        
+        var request = URLRequest(url: profileURL)
+        request.setValue("Bearer \(user.accessToken)", forHTTPHeaderField: "Authorization")
```

Comment:
I would rather prefer concatenation to interpolation in case someone for whatever reason decides to make `accessToken` optional

Reply:
done ✅